src/creature_senses.c

```diff
@@ -744,11 +758,19 @@ TbBool line_of_sight_3d(const struct Coord3d *frpos, const struct Coord3d *topos
         if (point_in_map_is_solid(&nextpos)) {
             SYNCDBG(7, "Player cannot see through (%d,%d) due to linear path solid flags (downcount %d)",
                 (int)nextpos.x.stl.num,(int)nextpos.y.stl.num,(int)distance);
+
+            JUSTMSG("Player cannot see through (%d,%d) due to linear path solid flags (downcount %d)",
```

Comment:
I'll test with this in for now, but we shouldn't forget to remove this before we merge.
Just like the one a few lines below

Reply:
Right.